Modules/Scripted/SampleData/SampleData.py

```diff
@@ -157,16 +158,22 @@ def __init__(self, sampleName=None, sampleDescription=None, uris=None, fileNames
         :param checksums: Checksum(s) formatted as ``<algo>:<digest>`` to verify the downloaded file(s). For example, ``SHA256:cc211f0dfd9a05ca3841ce1141b292898b2dd2d3f08286affadf823a7e58df93``.
         :param loadFiles: Boolean indicating if file(s) should be loaded.
         :param customDownloader: Custom function for downloading.
-        :param loadFileType: file format name(s) ('VolumeFile' by default if node name is specified).
+        :param loadFileTypes: file format name(s) (if not specified then the default file reader will be used).
         :param loadFileProperties: custom properties passed to the IO plugin.
+        :param loadFileType: deprecated, use ``loadFileTypes`` instead.
         """
+
+        # For backward compatibility (allow using "loadFileType" instead of "loadFileTypes")
+        if (loadFileType is not None) and (loadFileTypes is None):
+            loadFileTypes = loadFileType
```

Comment:
I suggest to raise an exception[^1] if both `loadFileType` and `loadFileTypes` are specified. Updated code should only use `loadFileTypes`

[^1]: I suggest `ValueError`. See https://stackoverflow.com/questions/256222/which-exception-should-i-raise-on-bad-illegal-argument-combinations-in-python